src/DynamoCore/Models/PortModel.cs

```diff
@@ -150,11 +155,7 @@ public PortModel(PortType portType, NodeModel owner, PortData data)
 
             SetPortData(data);
 
-            if (PortType == Models.PortType.Input)
-                MarginThickness = new Thickness(0);
-            else
-                MarginThickness = new Thickness(0, data.VerticalMargin, 0, 0);
-
+            MarginThickness = new Thickness(0);
```

Comment:
`MarginThickness` is retained for other references and will be removed completely in a separate work.